ext/curl/tests/curl_getinfo_CURLINFO_HTTPAUTH_USED-caddy.phpt

```diff
@@ -0,0 +1,34 @@
+--TEST--
+curl_getinfo - CURLINFO_HTTPAUTH_USED - online test
+--EXTENSIONS--
+curl
+--SKIPIF--
+<?php
+$curl_version = curl_version();
+if ($curl_version['version_number'] < 0x080c00) die("skip: test works only with curl >= 8.12.0");
+?>
+--FILE--
+<?php
+
+include 'skipif-nocaddy.inc';
```

Comment:
I think this needs to be moved to the `--SKIPIF-- section.

Reply:
Thank you, done now.